adding Checking keys in (#5920)
unlock-protocol/unlock: unlock-protocol.com/blog/checking-key-in.md

---

> title: Checking keys in

Reviewer: ```suggestion
title: Checking Keys In
```

subTitle: Tickets to conferences are memberships!
authorName: Julien Genestoux
publishDate: February 6, 2020

> description: At Unlock, we're proud to power the tickets for several conferences... but no ticketing solution is complete without a checking mechanism!

Reviewer: ```suggestion
description: At Unlock, we're proud to power the tickets for several conferences... but no ticketing solution is complete without a check-in mechanism!
```

image: /static/images/blog/introducing-tickets/introducing-tickets.jpg
---

> Memberships are everywhere: from your local gym, to your favorite news site, through conferences! At Unlock, we take great care in making sure we can _bridge_ the online memberships and the offline world.

Reviewer: ```suggestion
Memberships are everywhere: from your local gym, to your favorite news site, through conferences! At Unlock, we take great care in making sure we can _bridge_ the gap between online memberships and the offline world.
```

<p style="text-align:center" ><img width="300" src="/static/images/blog/checking-key-in/ethcc-ticket-qrcode.png" alt="qr code ticket"></p>

# Checking users in

Unlock keys are, by default, transferable. This means that, unless we use a mechanism to prevent this, someone could use their key (and the corresponding QR code) to enter a venue, transfer the key to another user, who would re-use the same key (and another version of the QR code) to enter the venue as well. **This is not how tickets are supposed to work**!

The verification page above works from any web browser... but, if viewed from inside a browser which has a crypto wallet and if the viewer is actually the lock owner, **the viewer can "mark" the ticket as checked-in**. For this, we are using metadata on the NFT which can only be set by the lock owner. This way, if the ticket is then transfered to someone else, the organizers would still be able to quickly identify sneaky users who are trying to re-use a ticket.

<p style="text-align:center" ><img width="300" src="/static/images/blog/checking-key-in/checked-in-key.png" alt="checked-in!"></p>

> With our support for credit cards, the ability to ask users to submit information for their tickets, and sell them on the organizers' website directly, we believe Unlock now provides a **tickecting experience** that's even better than most legacy solutions, while still leveraging Ethereum's core capabilities: decentralization, permissionless and programmable!

Reviewer: ```suggestion
With our support for credit cards, the ability to ask users to submit information for their tickets, and sell them on the organizers' website directly, we believe Unlock now provides a **ticketing experience** that's even better than most legacy solutions, while still leveraging Ethereum's core capabilities: decentralization, permissionless and programmable!
```
